### ISSUE TITLE
Responsive Orders table

### ISSUE THREAD

**Issue by alfetopito** [gnosis/gp-ui#693]
**Description**

Make the Orders table responsive, as per design

**Proposal**

![screenshot_2021-09-27_14-15-39](https://user-images.githubusercontent.com/43217/134986379-8e5e8f7b-c65c-4a23-b13f-54be50f7669b.png)
(I don't know how to disable the red lines :facepalm:)

https://www.figma.com/file/m7obOydxAhGNmlFYvxTk3Z/GP-Explorer?node-id=280%3A1

**Comment by elena-zh** [comment]
I noticed that there is no date on an order details in a mobile view. 
Are we OK with this?
I have to notice, that in the Cowswap sidebar we always show 'valid to' date (open/cancelled/expired) and 'filled on' (filled) for orders.
![image](https://user-images.githubusercontent.com/70885163/135069182-e29a100d-5c2a-44b7-8b47-06a2f84c8f62.png)

**Comment by alfetopito** [comment]
Good point Elena.
As mentioned during the call, @alongoni will work on that.
I'd start with only creation date. Execution date is not yet available from the API AFAICT.

In the mean time if someone starts working on it just re-use the same component for the date like what's is used in other places on the Explorer:

`Creation date           xxx xxx xxx`

**Comment by alongoni** [comment]
I've updated the design (the "creation date / created" column was added during the implementation 😅)
![UI_Explorer v1 - User details page-_ MOBILE](https://user-images.githubusercontent.com/622217/135158123-46b310d9-91a4-4216-b140-76c1df61a373.png)
)

**Comment by elena-zh** [comment]
@alongoni , thank you!
As far as I understand, we will show a tooltip with the full date adjusted to the local time zone when click on the date. Correct?

**Comment by alongoni** [comment]
> @alongoni , thank you! As far as I understand, we will show a tooltip with the full date adjusted to the local time zone when click on the date. Correct?

Right! I'm working improving this things and I'll sync with @matextrem in order to have all ready to adapt the design,
